DOL.WHD.Section14c.Web/src/modules/pages/dashboardTableConfig.js

```diff
@@ -1,4 +1,4 @@
-var employeeColumns = [
+var applicationColumns = [
   {
       "className": 'control',
```

Comment:
More double quotes, but in this case you don't need quotes at all.  Maybe I should spend some time with the linter config and figure out what its deal is - it's only finding problems in the tests.